charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.16.2
+version: 6.16.3
```

Comment:
Now should be 6.16.4.

Reply:
Updated to 6.16.6 (currently the next version)